src/plugins/intel_npu/src/plugin/npuw/partitioning/partitioning.cpp

```diff
@@ -1571,7 +1584,8 @@ void Partitioner::optimize(const std::string& func_name) {
         ctx.pmm_dims = cfg.get<::intel_npu::NPUW_PMM>();
         mergeParallelMatMuls(f._model, ctx);
 
-        // Concatenate closures for "concatenated" parameters
+        // Mark LazyTensors to be concatenated later
+        // Note: closures are properly processed later as well
         ov::ParameterVector new_params;
```

Comment:
I think these comments don't need to be updated. The transformation flow doesn't change here much in case of LazyTensors

Reply:
Removed